myapp/data/spotifytoken.go

```diff
@@ -147,11 +147,31 @@ func (t *SpotifyToken) Upsert(spotifytoken SpotifyToken) (int, error) {
 	spotifytoken.UpdatedAt = time.Now()
 
 	res2, err2 := collection.Insert(spotifytoken)
```

Comment:
We don't need to add numbered errors, just reuse err!

Reply:
✔️ will do